opentelemetry-proto/src/proto.rs

```diff
@@ -46,14 +46,14 @@ pub(crate) mod serializers {
 
     // AnyValue <-> KeyValue conversion
     pub fn serialize_to_value<S>(value: &Option<AnyValue>, serializer: S) -> Result<S::Ok, S::Error>
-        where
-            S: Serializer,
+    where
+        S: Serializer,
     {
         // Serialize any_value::Value using its own implementation
         // If value is None, it will be serialized as such
         match value {
             Some(any_value) => match &any_value.value {
-                Some(Value::IntValue(i)) => serialize_i64_to_string(i, serializer),
+                //Some(Value::IntValue(i)) => serialize_i64_to_string(i, serializer),
```

Comment:
@ramgdev does commenting it out resolve the export issue? From my understanding, [proto i64/u64 should be converted to JSON strings](https://protobuf.dev/programming-guides/proto3/#json)."

Reply:
@lalitb yes, this change fixes the exporter issue in the example. I see this in the JSON mapping doc ```JSON value will be a decimal string. Either numbers or strings are accepted.```  Since using numbers in serialization help with deserializing back into either standalone value or value wrapped in `opentelemetry::Value` enum, this should work imho.

Reply:
Not sure if I am looking at the wrong place. Looks like the attribute spec is asking `int64` to be encoded as number, not string. https://opentelemetry.io/docs/specs/otel/common/#attribute

Reply:
Doesn't that introduce ambiguity for deserializer without the explicit use of `Value::String` during serialization?

Reply:
resolved by https://github.com/open-telemetry/opentelemetry-rust/pull/1882